packages/skia/src/renderer/Canvas.tsx

```diff
@@ -76,7 +76,12 @@ export const Canvas = forwardRef(
     }, []);
 
     // Root
-    const root = useMemo(() => new SkiaSGRoot(Skia, nativeId), [nativeId]);
+    const root = useMemo(() => {
```

Comment:
I'm surprised this change helps, do you have examples I can try where I could see a difference there?

Reply:
Funnily enough, removing either set of changes (this one or the synchronous presentation) makes the bug return (i.e. the first frame renders empty).

Reply:
Happy to provide an example, but not sure what would be the best form? Ideally I would prepare a unit test demonstrating the issue, but I didn't find a suitable example to start off with – I would need to render a component, preferrably with frame-by-frame control, would appreciate hints on how to do it.